ecommerce/customers/serializers.py

```diff
@@ -0,0 +1,73 @@
+from django.db.transaction import atomic
+from rest_framework import serializers
+
+from customers.models import Customer, Address, Country, City
+
+
+class CustomerSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Customer
+        read_only_fields = ['id', 'created_at', 'updated_at']
```

Comment:
zaten read only degiller mi?

Reply:
default db'in bizim i.çin oluştuduğu alanlar read only olarak geldiğini bilmiyordum hocam.  Db'in bizim için ilgilendiği alanları o yüzden read only fields'a atmak istedim. Teşekkürler geri bildiirm için.